SCClassLibrary/Common/GUI/PlusGUI/Core/extGuiHistory.sc

```diff
@@ -0,0 +1,20 @@
++ History {
+
+	*makeWin { |where, numItems=8| ^current.makeWin(where, numItems) }
+
+	makeWin { |where, numItems=8|
+		var gui = HistoryGui(this, numItems);
+		if (where.notNil) { gui.moveTo(where.x, where.y) };
+		^gui
+	}
+
+	*document { current.document }
+
+	document { |title = ""|
+		var docTitle = title ++ Date.getDate.format("%Y-%m-%d-%Hh%M-History");
+		Document.new(docTitle, this.storyString)
+		// path not working yet
+		//.path_(docTitle); // don't lose title.
```

Comment:
are these comments stil relevant?

Reply:
I guess yes - Document:path_ still gives a NotYetImplemented error. would be noce to be able to set a path where this document would be saved when you hit save, rather than having to navigate and type in the path. 
hm, could write it as a file first, and then read in the file...
I'll think about that some more.

Reply:
Tried again, setting path still does not work, 
and IDE save dialog does not take the document title...
so I would keep the comment like this:
		//.path_(docTitle); //  path not working yet in Qt
		// - would be nice to set and show a path
		// where the doc can be saved later, by hand or automatically.

Reply:
thanks! in the future, would be good to mark that with `TODO` or `FIXME` so it can be easily found in the codebase